main.py

```diff
@@ -29,7 +30,7 @@ def main(argv):
     append_testcases(prefix, testsuite)
 
     xml_tree = ET.ElementTree(testsuites)
-    xml_tree.write("test_results.xml")
+    xml_tree.write(os.getcwd() + '\\' + 'test_results.xml')
```

Comment:
Let's create a local, `xml_file_path`, and assign it using `os.path.join(os.getcwd(), 'test_results.xml')`. That'll ensure that this can run regardless of the operating system that we're running the code on.

Reply:
I've made changes you've requested, would you like to check it out